src/twisted/python/failure.py

```diff
@@ -417,6 +383,14 @@ def __init__(self, exc_value=None, exc_type=None, exc_tb=None, captureVars=False
             )
             tb = tb.tb_next
 
+    @deprecatedProperty(Version("Twisted", 24, 8, 0))
```

Comment:
```suggestion
    @deprecatedProperty(Version("Twisted", "NEXT", 0, 0))
```

The trunk documentation is already updated to talk about `incremental` integration

https://docs.twisted.org/en/latest/development/compatibility-policy.html